packages/react-core/rollup.config.js

```diff
@@ -0,0 +1,31 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import scss from 'rollup-plugin-scss';
+import replace from '@rollup/plugin-replace';
+import { terser } from 'rollup-plugin-terser';
+
+const isProduction = process.env.IS_PRODUCTION;
+
+module.exports = {
+  input: 'dist/esm/index.js',
+  output: {
+    file: `dist/umd/react-core${isProduction ? '.min' : ''}.js`,
+    format: 'umd',
+    name: 'PatternFlyReact',
+    globals: {
+      react: 'React',
+      'react-dom': 'ReactDOM',
+      'prop-types': 'PropTypes'
+    }
+  },
+  external: ['react', 'react-dom', 'prop-types'],
```

Comment:
Didn't you remove prop-types references in another PR?

Reply:
Yes, but `prop-types` are still generated from the `typescript-to-proptypes` plugin. It seemed good to include these in our UMD bundle because there's no type support when using the `PatternflyReact` object.